workflows/wf_theiacov_illumina_pe.wdl

```diff
@@ -244,16 +266,19 @@ workflow theiacov_illumina_pe {
     String? pangolin_docker = pangolin4.pangolin_docker
     String? pangolin_versions = pangolin4.pangolin_versions
     # Clade Assigment
-    String nextclade_json = select_first([nextclade_one_sample.nextclade_json,""])
-    String auspice_json = select_first([ nextclade_one_sample.auspice_json,""])
-    String nextclade_tsv = select_first([nextclade_one_sample.nextclade_tsv,""])
-    String nextclade_version = select_first([nextclade_one_sample.nextclade_version,""])
-    String nextclade_docker = select_first([nextclade_one_sample.nextclade_docker,""])
-    String nextclade_ds_tag = select_first([abricate_flu.nextclade_ds_tag, nextclade_dataset_tag,""])
-    String nextclade_aa_subs = select_first([nextclade_output_parser_one_sample.nextclade_aa_subs,""])
-    String nextclade_aa_dels = select_first([nextclade_output_parser_one_sample.nextclade_aa_dels,""])
-    String nextclade_clade = select_first([nextclade_output_parser_one_sample.nextclade_clade,""])
-    String? nextclade_lineage = nextclade_output_parser_one_sample.nextclade_lineage
+    String nextclade_json = select_first([nextclade_one_sample_run1.nextclade_json,""])
+    String auspice_json = select_first([ nextclade_one_sample_run1.auspice_json,""])
+    String nextclade_tsv = select_first([nextclade_one_sample_run1.nextclade_tsv,""])
+    String nextclade_version = select_first([nextclade_one_sample_run1.nextclade_version,""])
+    String nextclade_docker = select_first([nextclade_one_sample_run1.nextclade_docker,""])
+    String nextclade_ds_tag = select_first([abricate_flu.nextclade_ds_tag_ha, nextclade_dataset_tag,""])
+    String nextclade_aa_subs = select_first([nextclade_output_parser_one_sample_run1.nextclade_aa_subs,""])
+    String? nextclade_tamiflu_aa_subs = select_first([nextclade_output_parser_one_sample_run1.nextclade_tamiflu_aa_subs,""])
+    String nextclade_aa_dels = select_first([nextclade_output_parser_one_sample_run1.nextclade_aa_dels,""])
+    String nextclade_clade = select_first([nextclade_output_parser_one_sample_run1.nextclade_clade,""])
+    String? nextclade_lineage = nextclade_output_parser_one_sample_run1.nextclade_lineage
+    # NA specific column - tamiflu mutation
+    String? tamiflu_resistance_aa_subs = nextclade_output_parser_one_sample_run2.nextclade_tamiflu_aa_subs
```

Comment:
Nice! So if I've read this correctly, we're currently able to run Nextclade a second time for flu samples using the NA segment against those specific dataset tags. And then filtering those results to output the subs associated with tamiflu resistance. This looks great!

I'm wondering if it will also be helpful to, at the workflow level, capture all of the NA sub/del outputs. Will need to consult with @emily-smith1 on if this is in alignment with PHL requests.

If so, we could probably just concatenate the `run1` and `run2` parser outputs and have that be the first option in these `select_first()` arrays, e.g.

```
153    call taxon_ID.nextclade_output_parser_one_sample as nextclade_output_parser_one_sample_run2 {
154      input:
155      nextclade_tsv = nextclade_one_sample_run2.nextclade_tsv,
156      organism = organism,
157      NA = true
158    }
159   na_nextclade_ds_tag= "~{abricate_flu.nextclade_ds_tag_ha + ',' + abricate_flu.nextclade_ds_tag_na}"
160   na_nextclade_aa_subs= "~{nextclade_output_parser_one_sample_run1.nextclade_aa_subs + ',' + nextclade_output_parser_one_sample_run2.nextclade_aa_subs}"
161   na_nextclade_aa_dels= "~{nextclade_output_parser_one_sample_run1.nextclade_aa_dels + ',' + nextclade_output_parser_one_sample_run2.nextclade_aa_dels}"

...

 
273    String nextclade_docker = select_first([na_nextclade_ds_tag,nextclade_one_sample_run1.nextclade_docker]) 
274    String nextclade_aa_subs = select_first([na_nextclade_aa_subs, nextclade_output_parser_one_sample_run1.nextclade_aa_subs])
275    String? nextclade_tamiflu_aa_subs = select_first([nextclade_output_parser_one_sample_run1.nextclade_tamiflu_aa_subs,""])
276    String nextclade_aa_dels = select_first([na_nextclade_aa_dels,nextclade_output_parser_one_sample_run1.nextclade_aa_dels])
```

Reply:
Yes, we should have an output column that captures all of the NA substitutions and deletions. The NA gene is a major virulence factor and it would be important to capture mutations that have impacts beyond just antiviral resistance.

Reply:
All AA substitutions and deletions be they for HA or NA are always retained in their respective columns as they come off of Nextclade. In fact the tamiflu resistance mutations are extracted from the NA AA substitutions column.

Reply:
```
    # NA specific columns - tamiflu mutation
    String? tamiflu_resistance_aa_subs = nextclade_output_parser_one_sample_run2.nextclade_tamiflu_aa_subs
    String nextclade_na_aa_subs = select_first([nextclade_output_parser_one_sample_run2.nextclade_aa_subs,""])
    String nextclade_na_aa_dels = select_first([nextclade_output_parser_one_sample_run2.nextclade_aa_dels,""])
  ```
  Added the following columns to the output of the workflow in commit https://github.com/theiagen/public_health_viral_genomics/pull/214/commits/413313cc6b1b7fece92dfeb7eb552837caf9e172. It's not very elegant to have a column named `nextclade_aa_subs` for the HA segment, and `nextclade_na_aa_subs` for the NA segment, but renaming the first column will affect the results table for other organisms such as SC2. 
  
  Tested both locally and [on Terra](https://app.terra.bio/#workspaces/theiagen-validations/Theiagen_Mendes_Sandbox/job_history/9bb157a3-1707-48c3-87d2-ced2aa6af153)
  
<img width="518" alt="image" src="https://user-images.githubusercontent.com/15690332/220637188-445edde2-6223-4f33-83de-4f99e3bceba8.png">

Reply:
I just tested this branch successfully on 38 flu sequences from CDPH. However, it's confusing that there is a nextclade_aa_subs column, then a nextclade_na_aa_subs column. Could the former be modified to be nextclade_ha_aa_subs to clearly distinguish which columns contain substitutions from which gene?

Reply:
Thanks @emily-smith1 and @kevinlibuit! In commit https://github.com/theiagen/public_health_viral_genomics/pull/214/commits/4fcd707d0b941144a7108b12e2431104c494ffb6 I've added the suggested alteration to concatenate into the `nextclade_aa_subs`, `nextclade_aa_dels` and `nextclade_ds_tag` both HA and NA information when the "flu" organism is selected. Tested locally and [on Terra](https://app.terra.bio/#workspaces/theiagen-validations/Theiagen_Mendes_Sandbox/job_history/c6e3d524-185c-411a-abac-088be5a77349).
<img width="527" alt="image" src="https://user-images.githubusercontent.com/15690332/220718847-615dd7a8-60da-46bb-a5c3-13b88cd26ae2.png">